<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Room Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>671</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string> Create room</string>
    </property>
    <widget class="QPushButton" name="createBtn">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>105</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="roomName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Room name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="roomDesc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Room Description</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>171</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Available Rooms </string>
    </property>
    <widget class="QListWidget" name="availableRoomsListWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>161</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>471</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Information Panel</string>
    </property>
    <widget class="QPlainTextEdit" name="infoPanelTextBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>471</width>
       <height>251</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Click on the room to get information about it.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
